### ISSUE TITLE
plot_posterior_predictive(original_scale=True) DTypePromotionError

### ISSUE THREAD

**Issue by patrickgamer** [pymc-labs/pymc-marketing#976]
I'm successfully running `MMM.sample_posterior_predictive(X, extend_idata=True)` but when I call `plot_posterior_predictive(original_scale=True)` I get a type error:

`DTypePromotionError: The DType <class 'numpy.dtypes.DateTime64DType'> could not be promoted by <class 'numpy.dtypes.Float64DType'>. This means that no common DType exists for the given inputs. For example they cannot be stored in a single array unless the dtype is `object`. The full list of DTypes is: (<class 'numpy.dtypes.DateTime64DType'>, <class 'numpy.dtypes.Float64DType'>)`

Basically it appears that `MMM` is trying to combine my 'Date' data with my Y values. It's weird b/c I would expect if there was any misunderstanding about how the Date column was being used, it'd appear in the sampling. However sampling appears to run just fine.

My guess (I'm pretty new to pymc) is that there is a problem specifically within plotting feature where it's not handling default date column properly.

PyMC version: 5.15.1
PyMC Marketing version: 0.8.0

**Comment by juanitorduz** [comment]
Thanks, @patrickgamer. Could you please provide us with a minimal reproducible example where we can see this error? This will help us find the issue. Thanks!

**Comment by anevolbap** [comment]
`plot_posterior_predictive` was removed with the old MMM class in #2487. OK to close?
